FlowCrypt/Controllers/Attachment/AttachmentViewController.swift

```diff
@@ -99,22 +131,40 @@ extension AttachmentViewController: WKNavigationDelegate {
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
         hideSpinner()
     }
+
+    func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction) async -> WKNavigationActionPolicy {
+        guard
+            navigationAction.request.url?.scheme == "data",
+            !isNavigated
+        else { return .cancel }
+
+        isNavigated = true
+        return .allow
+    }
 }
 
 private extension AttachmentViewController {
 
+    private func setContentRules() async throws {
+        guard let list = try await WKContentRuleListStore.default().compileContentRuleList(
+            forIdentifier: "blockRules",
+            encodedContentRuleList: encodedContentRules
+        ) else {
+            throw AppErr.general("Could not produce a data URL to preview this file")
+        }
+
+        webView.configuration.userContentController.removeAllContentRuleLists()
```

Comment:
Is it necessary to remove all previous content rules lists?
What if user has some content-blocker installed with custom block lists, will they be removed too?

Reply:
Not sure, but I'd rather we have more control, until the details are more understood. Which would mean remove all first.